interface/forms/fee_sheet/new.php

```diff
@@ -1341,8 +1341,12 @@ function newEvt() {
 
 if ($GLOBALS['contract_physician_in_feesheet']) {
   echo "&nbsp;&nbsp;" . xlt($GLOBALS['contract_physician_in_feesheet_name']) . "\n";
+  if (trim($GLOBALS['contract_physician_in_feesheet_name']) == 'Users') {
```

Comment:
I'm going to have to figure out how to convert my system to use this efficiently.  ARSP has that "counselor_id" thing where it checks pay rates and all that.  I will try to help blend the functions in the near future.